scripts/perf_to_stats_csv.jq

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env -S jq -Mf -s -r
+
+[ 
+  .[] |
+  select(has("stats")) |
+  .stats |
+  {
+    sdkRevision: (.metadata.testData.sdkRevision | tostring), 
+    sdkCommitTime: (.metadata.testData.sdkCommitTime | strftime("%Y-%m-%d %H:%M:%S")),
```

Comment:
What timezone is the sdkCommitTime measured in?  I'd prefer UTC, if possible.

Reply:
This is plumbed from the `git show` command producing a Unix timestamp:
https://github.com/Agoric/testnet-load-generator/blob/d8c3e286349ca43d7ac64aac1a3bfb1bf7109295/start.sh#L37

And according to [`jq`'s manual](https://stedolan.github.io/jq/manual/#Dates), `strftime` formats using UTC/GMT: 
> The `strftime(fmt)` builtin formats a time (GMT) with the given format. The `strflocaltime` does the same, but using the local timezone setting.